packages/world/src/modules/keyswithvalue/KeysWithValueHook.sol

```diff
@@ -20,15 +20,19 @@ import { getTargetTableSelector } from "../utils/getTargetTableSelector.sol";
  *
  * Note: if a table with composite keys is used, only the first key is indexed
  */
-contract KeysWithValueHook is IStoreHook {
+contract KeysWithValueHook is StoreHook {
   using ArrayLib for bytes32[];
   using ResourceSelector for bytes32;
 
+  function _world() internal view returns (IBaseWorld) {
+    return IBaseWorld(storeAddress(msg.sender));
+  }
+
```

Comment:
this is just for consistency, I'm not sure how hooks should handle this, but it wouldn't break things to just use `msg.sender` and optimize `storeAddress` away